hub/core/partial_sample.py

```diff
@@ -0,0 +1,21 @@
+from typing import Tuple, Optional, Union
+import numpy as np
+
+
+class PartialSample(object):
```

Comment:
can remove the inheritance from object

Reply:
also, a docstring explaining when this class is used might be helpful for reference